v0/destinations/googlesheets/transform.js

```diff
@@ -87,61 +88,77 @@ const processWithCustomMapping = (message, attributeKeyMapping) => {
   return responseMessage;
 };
 
-// Main process Function to handle transformation
-const process = event => {
+const batch = events => {
+  return { batch: events };
+};
+
+// Function for transforming single payload
+const processSingleMessage = event => {
   const { message, destination } = event;
   if (destination.Config.sheetName) {
     const payload = {
-      message: processWithCustomMapping(
-        message,
-        destination.Config.eventKeyMap
-      ),
-      spreadSheetId: destination.Config.sheetId,
-      spreadSheet: destination.Config.sheetName
+      message: processWithCustomMapping(message, destination.Config.eventKeyMap)
     };
     return payload;
   }
   throw new CustomError("No Spread Sheet set for this event", 400);
 };
 
+// Main process Function to handle transformation
+const process = event => {
+  const payload = processSingleMessage(event);
+  const batchedResponse = batch(payload);
```

Comment:
Can you add some documentation here as to why the returned response is wrapped in a batch object?

Reply:
Added inline comment explaining the reason for this